src/runtime/vulkan/vulkan_stream.cc

```diff
@@ -55,11 +55,15 @@ VulkanStream::VulkanStream(const VulkanDevice* device)
   cb_begin.flags = VK_COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT;
   cb_begin.pInheritanceInfo = 0;
   VULKAN_CALL(vkBeginCommandBuffer(state_->cmd_buffer_, &cb_begin));
+
+  profiler_ = new AmdRgpProfiler(device_);
```

Comment:
Please do this only when RGP is enabled.

Reply:
> Please do this only when RGP is enabled.

Okay.  Will Implement your suggestion.